genrl/deep/agents/vpg/vpg.py

```diff
@@ -56,6 +56,7 @@ def __init__(
         gamma: float = 0.99,
         epochs: int = 1000,
         lr_policy: float = 0.01,
+        lr_value: float = 0.5,
```

Comment:
Lr value should not be needed in VPG, I think

Reply:
I'll need to change it.